static/app/views/replays/detail/console/consoleLogRow.tsx

```diff
@@ -151,6 +147,3 @@ const Message = styled('div')`
   white-space: pre-wrap;
   word-break: break-word;
 `;
-
-const ConsoleLogRow = memo(UnmemoizedConsoleLogRow);
-export default ConsoleLogRow;
```

Comment:
Also dropped this extra memo() call, it's unique in all the replay tabs.. we don't need it.